Automattic/sensei
# Add content drip email to be sent to student when lesson becomes available

Resolves https://github.com/Automattic/sensei/issues/6472

Depends on sensei-pro branch https://github.com/Automattic/sensei-pro/pull/2169

## Proposed Changes
* Added email pattern for content drip email

## Testing Instructions

1. Enable the Email Customization feature
2. Checkout this branch of sensei-pro https://github.com/Automattic/sensei-pro/pull/2169
3. Re-Run the email generator tool
4. Create a course
5. Add a lesson
6. Go to the lesson editor
7. In the content drip settings below the editor, select "On a specific date", and select today's date
![image](https://user-images.githubusercontent.com/6820724/223964988-7d113218-ac12-4215-be8c-c08dcdbd57cc.png)
8. Send a test email clicking the button below
9. Make sure the target address got an email as expected
10. Take the course as a student
11. Install WP Control plugin
12. Go to Tools -> Cron Events
13. Find the hook `woo_scd_daily_cron_hook`
![image](https://user-images.githubusercontent.com/6820724/223965679-3c792dbe-9a75-48ff-a3a1-7ab83a9b18b9.png)
14. You have click on 'Run Now' below the hook name, but before that you have to comment out the line below, otherwise it will force the event to run only once at specific time and Run Now button won't work, because it will reset the next run time https://github.com/Automattic/sensei-pro/blob/3768080e0f89b31bfa5f58b09d46b9f82fc8bc55/modules/content-drip/includes/class-sensei-content-drip.php#L139
15. Now click on run now
16. Make sure the student got an email as expected

![image](https://user-images.githubusercontent.com/6820724/223967820-e724dbdd-e4bb-43f4-afbb-c261b214fafb.png)

## Pre-Merge Checklist
